source/userguide/entity_resolver.rst

```diff
@@ -210,10 +210,42 @@ Upon building all the required resolvers, to access a particular resolver, adapt
 
   <ElasticsearchEntityResolver ready: True, dirty: False>
 
-Access & train an entity resolver as standalone component
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) Access & train specific entity resolvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Accessing entity resolvers and training them indepenedent of the :mod:`nlp` pipeline is sometimes required, for example, to evaluate their performances on your dataset(s). When you are ready to begin experimenting, import `EntityResolverFactory` as follows:
+As an alternative to building all entity resolvers required for an app, you can also access and train resolvers of a particular domain-intent hierarchy. This might be useful in case you want to to run some simple tests for specific resolvers.
```

Comment:
```suggestion
As an alternative to building all entity resolvers required for an app, you can also access and train resolvers of a particular domain-intent hierarchy. This might be useful in case you want to run some simple tests for specific resolvers.
```